docs/apps/cuda-quantum.md

```diff
@@ -0,0 +1,98 @@
+---
+tags:
+  - Free
+---
+
+# CUDA Quantum
+
+CUDA Quantum is a single-source, modern C++ programming model and compiler platform 
+for the quantum acceleration of existing heterogeneous computing architectures.
+
+!!! info "News"
+     **27.7.2023** Installed `cuda-quantum/0.4.0`.
+
+## Available
+
+Currently supported Qiskit versions:
+
+| Version | Module               | Puhti | Mahti | Notes           |
+|:--------|:---------------------|:-----:|:-----:|-----------------|
+| 0.4.0  | `cuda-quantum/0.4.0`  | X     | X     | default version |
+
+All modules are based on containers using Apptainer (previously known as Singularity).
+Wrapper scripts have been provided so that common commands such as `python`,
+`python3`, `pip` and `pip3` should work as normal. For more information, see
+[CSC's general instructions on how to run Apptainer containers](../computing/containers/run-existing.md).
+
+## License
+
+CUDA Quantum is licensed under
+[Apache License 2.0](https://github.com/Qiskit/qiskit-metapackage/blob/master/LICENSE.txt).
+
+## Usage
+
+To use the default version of CUDA Quantum on Puhti or Mahti, initialize
+it with:
+
+```text
+module load cuda-quantum
+```
```

Comment:
Please leave one line empty after a code block. This is to prevent MkDocs from leaving the following paragraph, in this case "If you wish to have...", as loose text, i.e. outside of an HTML `<p>`.